Command line features: Create a user & more install options

ClaraLeigh opened the pull request:
## Features

1. Add colors to the install script to make it easier to read
2. Ask for the mongodb url
3. Ask if user registration should be disabled
4. Create new `npm run create-user` command to create a user via the command line

## Testing

This has been tested in mac & linux. I think from memory in windows you'll just miss the colors unless the terminal has been setup to work with it (not breaking)

## Install Update Screenshot
<img height="500" alt="image" src="https://github.com/danny-avila/LibreChat/assets/2524209/cc49cd6a-46ce-48b7-832b-4ad2cad9af00">

## Create User Screenshot
<img height="500" alt="image" src="https://github.com/danny-avila/LibreChat/assets/2524209/a0afd254-3d8f-4c45-a859-3897c3982386">

ClaraLeigh committed:
feature: Ask for mongodb url during the npm install

ClaraLeigh committed:
chore: Add warning if mongodb url looks incorrect in install

ClaraLeigh committed:
chore: Add color to the install script
- This will only work on terminals setup to use color

fuegovic reviewed (commented):
Worked great and with colors on Windows 11
![image](https://github.com/danny-avila/LibreChat/assets/32828263/cb23c827-869f-4bab-b76f-fb970c3fd2ca)

ClaraLeigh committed:
chore: Change package names to be meaningful
- Makes it easier to call them from the outside or even include the package in a different project (ie frontend only)

ClaraLeigh committed:
chore: Rename color-console.js to helper.js and add askQuestion as exported fn

ClaraLeigh committed:
feature: Create a user from the command line

ClaraLeigh committed:
chore: Add more comments and user messages to create-user cmd

fuegovic commented:
here's the updated color scheme on W11
![image](https://github.com/danny-avila/LibreChat/assets/32828263/2f003cf0-eb78-4621-9375-0c873edfd5ca)

ClaraLeigh committed:
bugfix(install): Handle missing .env.example
